deployments/container/Dockerfile

```diff
@@ -72,6 +65,9 @@ LABEL description="NVIDIA DRA Driver for GPUs"
 LABEL org.opencontainers.image.description="NVIDIA DRA Driver for GPUs"
 LABEL org.opencontainers.image.source="https://github.com/NVIDIA/k8s-dra-driver-gpu"
 
+# When doing a cross-platform build (e.g., amd64 -> arm64) then mkdir/mv below
```

Comment:
Could we cheat a little and use `WORKDIR` and `COPY --from=build`?

Something like:
```
WORKDIR /licenses
COPY --from=build /NGC-DL-CONTAINER-LICENSE /licenses/NGC-DL-CONTAINER-LICENSE
```
(This doesn't REMOVE the source file though)

Reply:
I think so! At first, I had the same idea (to not invoke mkdir/mv so that we don't have any `RUN` statement in this stage).

Then I thought: we will probably have to add _some_ `RUN` statement at some point to this build stage. So I concluded that "never have any RUN statement" is not the best goal follow. And instead it's good to add a bit of commentary on the topic now while it's fresh on our minds (reminding us that in a cross-platform build in _this_ stage all binaries run in a virtualized environment).

Reply:
This is straight copied from the Dockerfiles of the other repos. If we change it here we should probably change it there too.

Reply:
(I am advocating for not changing that)